app/src/main/res/values/strings.xml

```diff
@@ -388,6 +388,8 @@
     <string name="importer_help_totp_authenticator">Supply a TOTP Authenticator export file.</string>
     <string name="importer_help_winauth">Supply a WinAuth export file.</string>
 
+    <string name="importer_encrypted_exception_google_authenticator">Encrypted entry:</string>
```

Comment:
Could you change this to "Encrypted entry was skipped: %s" and pass the "email" as a parameter?